test/test_hybridsim.cpp

```diff
@@ -17,25 +17,58 @@ TEST(HybridSimTest, TrivialParallelTest) {
     };
 
     HybridSchrodingerFeynmanSimulator ddsim(quantumComputation(), HybridSchrodingerFeynmanSimulator::Mode::DD);
-    ddsim.Simulate(1);
+
+    auto resultDD = ddsim.Simulate(8192);
+    for (const auto& entry: resultDD) {
+        std::clog << "resultDD[" << entry.first << "] = " << entry.second << "\n";
+    }
```

Comment:
Are these logs a remainder of the debugging process or are they intended to stay?

Reply:
I'd keep them for debug output if theses tests happen to fail again. But I'll change the stream to `std::cout`.